src/main/java/org/gradle/profiler/ScenarioSettings.java

```diff
@@ -16,4 +21,27 @@ public InvocationSettings getInvocationSettings() {
     public GradleScenarioDefinition getScenario() {
         return scenario;
     }
+
+    public File getJfrProfilerOutputLocation() {
```

Comment:
Getters which are not getters :/ `computeJfrProfilerOutputLocation()`?

Reply:
Renamed.